fuzz/fuzz_targets/net.rs

```diff
@@ -144,9 +144,6 @@ fuzz_target!(|bytes| {
         vec![(0, input_queue, input_evt), (1, output_queue, output_evt)],
     )
     .unwrap();
-
-    // Wait for the events to finish and net device worker thread to return
-    net.wait_for_epoll_threads();
```

Comment:
The ordering here is really important. Since `VirtioCommon::wait_for_epoll_theads` is safe to call multiple times then I think the best approach is to keep these explicit extra calls.

Reply:
@phip1611 I've updated your PR to remove the second patch as it still needs consideration whereas the first patch is clearly good.